src/System.ObjectModel/src/System/Collections/ObjectModel/KeyedCollection.cs

```diff
@@ -45,6 +47,19 @@ protected KeyedCollection(IEqualityComparer<TKey> comparer, int dictionaryCreati
             _threshold = dictionaryCreationThreshold;
         }
 
+        /// <summary>
+        /// Enables the use of foreach internally without allocations using <see cref="List{T}"/>'s struct enumerator.
+        /// </summary>
+        new private List<TItem> Items
```

Comment:
Are there  any uses in the assembly that would hence benefit from this being `new internal`?